docs/src/pages/[platform]/connected-components/liveness/QuickStartReact.tsx

```diff
@@ -56,6 +56,13 @@ export function LivenessQuickStartReact() {
           sessionId={createLivenessApiData.sessionId}
           region="us-east-1"
           onAnalysisComplete={handleAnalysisComplete}
+          // These URL's are our recommended CDN's supported by Rekognition and will be set to default in the next Major Version
```

Comment:
I don't think we want to callout the major version in the code sample since it will quickly be out of date when customers upgrade.

```suggestion
          // Amazon Recognition CDN for WASM and model files
```